### PR TITLE
Add I/O related negative tests 

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2261

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/tests/negative.bal [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import ballerina/lang.'string as langstring;
+
+@test:Config {}
+isolated function testFileNotFound() {
+    string filePath = TEMP_DIR + "unknown.txt";
+    var err = openReadableFile(filePath);
+
+    if err is Error {
+        test:assertTrue(langstring:includes(err.message(), "no such file or directory:", 0));
+        test:assertTrue(langstring:includes(err.message(), "unknown.txt", 0));
```
Comment (daneshk) [inline ballerina/tests/negative.bal:26]:
```suggestion
        test:assertTrue(err.message().includes("no such file or directory:"));
        test:assertTrue(err.message().includes("unknown.txt"));
```

Comment (daneshk) [reply ballerina/tests/negative.bal:26]: We may also like the above easily with lang.string import

**Review by daneshk — DISMISSED** [review]
Add a minor suggestion. Overall LGTM

File: ballerina/tests/negative.bal (hunk 1 above)
Comment (BuddhiWathsala) [reply ballerina/tests/negative.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/400/commits/551dea9487bb7b9e1c9bd608aef5fdaf45fe21ed

**Review by daneshk — APPROVED** [review]

**Pull request merged**